test/prepare-unstable.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+setup() {
+  src_dir="${BATS_TEST_DIRNAME}/../src"
```

Comment:
Where is the BATS_TEST_DIRNAME variable coming from?

Reply:
It's provided by Bats https://github.com/bats-core/bats-core#special-variables